src/content/developers/docs/consensus-mechanisms/pos/faqs/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Frequently Asked Questions
+description: Frequently asked questions on proof-of-stake Ethereum.
+lang: en
+---
+
+## What is proof-of-stake {#what-is-proof-of-stake}
+
+Proof-of-stake is a class of algorithm that can provide security to blockchains by ensuring that assets of value are lost by attackers who act dishonestly. Proof-of-stake systems require a set of validators to make some asset available that can be destroyed if the validator engages in some provably dishonest behaviour. Ethereum uses a proof-of-stake mechanism to secure the blockchain.
+
+## How does proof-of-stake compare to proof-of-work? {#comparison-to-proof-of-work}
+
+Both proof-of-work and proof-of-stake are mechanisms that economically disincentivize malicious actors from spamming or defrauding the network. In both cases, nodes that actively participate in consensus put some asset "into the network" that they will lose if they misbehave.
+
+In proof-of-work this asset is energy. The node, known as a miner, runs an algorithm that aims to compute a value with a specific format faster than any other node. If they manage, they have the right to propose a block to the chain. In order to change the history of the chain, or dominate the block proposal, a miner would have to have so much computing power that they always win the race. The energy required to "mine" using proof-of-work is a real world asset that miners pay for.
+
+Proof-of-stake requires nodes, known as validators, to explicitly submit a crypto asset to a smart contract. If they misbehave then this crypto can be destroyed. This is a direct "staking" of assets directly into the chain, rather indirectly via energy expenditure.
+
+Proof-of-work is much more energy-hungry because electricity is burned in the mining process. Proof-of-stake, on the other hand, requires only a very small amount of energy - Ethereum validators cna even run on a low-powered device such as Raspberry Pi. Ethereum's proof-of-stake mechanism is thought to be more secure than proof-of-work because the cost to attack is greater, and the consequences to an attacker are more severe.
+
+Proof-of-work versus proof-of-stake has long been a hotly debated topic. A great summary of the arguments is provided in this [blog post](https://vitalik.ca/general/2017/12/31/pos_faq.html#what-are-the-benefits-of-proof-of-stake-as-opposed-to-proof-of-work) and this debate between Justin Drake and Lyn Alden:
+
+<iframe width="500" height="300" src="https://www.youtube.com/embed/1m12zgJ42dI" title="The PoW vs. PoS Debate | Lyn Alden & Justin Drake" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
your call - i prefer the embed.

Reply:
Cool. No biggie; we don't have a strict process in place yet, so let's just run with it. We should use the YouTube component here instead of an iframe embed though.